### DISCUSSION TITLE
How to use the new Algolia DocSearch Block

### DISCUSSION THREAD

**Discussion by vicam001** [lowdefy/lowdefy#1657] [Questions and Help] [answered]
Hi! 
I'm delighted with the recent addition of the DocSearch block https://docs.lowdefy.com/DocSearch, which should be available in lowdefy 4.0.0-rc.10, however when I try to add it into my app, I get the following error ( also having added the app.html.appendHead part) :

`
✖ Block type "DocSearch" was used but is not defined.
`

Could it be that it has not been added yet as an "official" plugin available within the default Lowdefy runtime ? May you please add it there so that it is usable without having to deal with it as an extra plugin ( I am afraid I am not able yet to make plugins work ) 

Many thanks !

**Comment by Gervwyk** [comment, marked as answer]
Hi @vicam001 

It will be available as a default block on the next release. The Lowdefy docs currently installs it out of the monorepo workspace, which you can also do if you add it as local plugin, but it will be available on next release.

**Reply by vicam001** [reply]
Thanks for the response, @Gervwyk , I very much look forward to that, DocSearch is a great addition to the Lowdefy ecosystem.
